cmd/scollector/main.go

```diff
@@ -149,7 +150,7 @@ func readConf() {
 
 func main() {
 	flag.Parse()
-	if *flagPrint || *flagDebug {
+	if *flagPrint || *flagDebug || *flagHTTP {
 		slog.Set(&slog.StdLog{Log: log.New(os.Stdout, "", log.LstdFlags)})
```

Comment:
The general idea is that most settings are available via flag **and** conf file. As it reads the config file it simply overrides the flag value if it finds that in the conf file. 

Generally all we need to do is add a flag and a matching case in `readConf()`.

Feel free to join us in our slack room. I sent you an invite.